tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -113,3 +113,13 @@ def test_c2r_latest_older_unsupported_version(convert2rhel):
     # Clean up
     os.system(f"cp /tmp/__init__.py {PATH_TO_VERSION}")
     del os.environ["CONVERT2RHEL_UNSUPPORTED_VERSION"]
+
+
+def test_clean_cache(convert2rhel):
+    # Test that the yum clean is done before any other check that c2r does
+    with convert2rhel("--no-rpm-va --debug") as c2r:
+        c2r.expect("Prepare: Clean yum cache metadata")
+        c2r.expect("Cached yum metadata cleaned successfully.")
+        c2r.expect("Prepare: Perform basic system checks")
+        c2r.expect("Continue with the system conversion?")
```

Comment:
```suggestion
        c2r.expect("Cached yum metadata cleaned successfully.")
        c2r.expect("Continue with the system conversion?")
```

Needs to be removed as it was removed in the main.py